adonis-typings/i18n.ts

```diff
@@ -185,6 +185,12 @@ declare module '@ioc:Adonis/Addons/I18n' {
      */
     provideValidatorMessages: boolean
 
+    /**
+     * Add this function if you want to have control over what is returned
+     * when an identifier is missing.
+     */
+    fallBackIfNoIdentifier?: (identifier: string, locale: string) => any
```

Comment:
I think we should ask that this function always return a `string` instead of `any`

Otherwise, it will break type safety on the `formatMessage` method